### ISSUE TITLE
Add Snyk Docker scan for the frontend Docker image

### ISSUE THREAD

**Issue by aerabi** [aerabi/events#26]

**Comment by Rajdeep1311** [comment]
Is the issue open?

**Comment by aerabi** [comment]
Yes, it is, @Rajdeep1311. You can also use Trivy or Docker Scout.
Good hunting.

**Comment by Rajdeep1311** [comment]
Sir, I raised a PR please do check @aerabi

**Comment by aerabi** [comment]
The PR only has updates in the Dockerfile's comments. The CI job didn't end up there. Maybe caused by git merge or something? 🤔

**Comment by Rajdeep1311** [comment]
But I added lines to Snyk scan the file sir @aerabi

**Comment by aerabi** [comment]
Where did you add it, @Rajdeep1311?
The [PR](https://github.com/aerabi/events/pull/43/files) has only changes to the Dockerfile.

**Comment by Rajdeep1311** [comment]
Oh okay sir, can you please suggest what could be done to scan the image sir?
